cmd/thor/flags.go

```diff
@@ -150,6 +150,15 @@ var (
 		Value: "localhost:2112",
 		Usage: "metrics service listening address",
 	}
+	enableAdminFlag = cli.BoolFlag{
```

Comment:
should we put it into documentation or these flags are hidden ?

Reply:
Sure I can add it

Reply:
Btw when looking into the docs I saw this:

Metrics is enabled in nodes by default. It's possible to disable it by setting  `--enable-metrics=false`.

Isn't this invalid as of now?

Reply:
Ya I think this is wrong

Reply:
Should it be fixed in this PR?

Reply:
No need to change. AFAIK the decision was made to leave metrics enabled per default given that the default server was bounded to localhost on a different port.

Reply:
So no documentation for the new flags? @otherview